apps/docs/src/resources/integrations/payments/paypal.md

```diff
@@ -157,6 +190,42 @@ In the example above we do a couple of things:
 
 This event is called when the user approves the payment process. It's the last step before the payment is captured.
 
+```ts
+  ...
+  // part of window.paypal.Buttons({}) params
+  onApprove: async (data: OnApproveData, actions: OnApproveActions) => {
+    // createOrder from useCheckout composable
+    orderCreated.value = await createOrder({
+      paypalOrderId: data.orderID,
+      // optional - can be acdc, applepay, googlepay or venmo
+      product: 'sbp',
+    });
+    refreshCart()
+    // apiInstance from useShopwareContext composable
+    const handlePaymentResponse = await apiInstance.invoke.post(
+      "/store-api/handle-payment",
+      {
+        orderId: orderCreated.value.id,
+        successUrl: `${window.location.origin}/order/success?order=${orderCreated.value.id}&success=true`,
+        errorUrl: `${window.location.origin}/order/success?order=${orderCreated.value.id}&success=false`,
```

Comment:
doesn't that need the `paypalOrderId` to capture the created order? Same below in the existing example, I believe this was always an issue that this was not documented.